src/GitUtils.ps1

```diff
@@ -453,7 +517,13 @@ function Find-Ssh($program = 'ssh-agent') {
 }
 
 # Loosely based on bash script from http://help.github.com/ssh-key-passphrases/
-function Start-SshAgent([switch]$Quiet) {
+function Start-SshAgent([switch]$Quiet, [string]$StartupType = 'Manual') {
```

Comment:
Could we add `[ValidateSet("Automatic", "Boot", "Disabled", "Manual", "System")]` to the `$StartupType` parameter?  This will enable tab-completion on this paramter.